src/UI/Pages/ServiceInventory/Components/TableControls/TableControls.tsx

```diff
@@ -45,7 +45,7 @@ export const TableControls: React.FC<Props> = ({
           identityAttribute={identityAttribute}
         />
         <ToolbarItem variant="separator" />
```

Comment:
Maybe this separator is not necessary anymore?
Either way, the distinction is clearer now :+1:

Reply:
Ah yes, I will remove the separator.